src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/types.js

```diff
@@ -121,12 +121,17 @@ export interface FormStoreInterface {
     +options: SchemaOptions,
     +resourceKey: ?string,
     +schema: Object,
+    +set: (name: string, value: mixed) => void,
     +setMultiple: (data: Object) => void,
     +setType: (type: string) => void,
     +types: {[key: string]: SchemaType},
     +validate: () => boolean,
 }
 
+export type FieldChangeContext = {
```

Comment:
maybe call this `FormChangeContext` and pass it to the `FormStoreInterface::change` method